docs/reference/rest_api_requests.md

```diff
@@ -0,0 +1,242 @@
+
+
+### Get a comment
+
+`GET /api/requests/{request_id}/comments/{comment_id}`
+
+**Parameters**
+
+| Name       | Type   | Location | Description                                                  |
+| ---------- | ------ | -------- | ------------------------------------------------------------ |
+| `request_id`   | string | path     | The request's public identifier. |
+| `comment_id`       | string | path     | The comment's public identifier                |
+| `accept`   | string | header   | - `application/json` (default)<br />- `application/vnd.inveniordm.v1+json` (not yet) |
+
+**Request**
+
+```http
+GET /api/requests/{request_id}/comments/{comment_id} HTTP/1.1
+Accept: application/json
+```
+
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "created": "2020-11-27 10:52:23.945755",
+  "id": "{comment_id}",
+  "links": {
+    "self": "{scheme+hostname}/api/requests/{rquest_id}/comments/{comment_id}"
+  },
+  "payload": {
+    "content": "I would use these subject terms to align the record with others in the community.",
+    "format": "html"
+  },
+  "revision_id": 1,
+  "type": "C",
+  "updated": "2020-11-27 10:52:23.969244",
+}
+```
+
+### Update a comment
+
+`PUT /api/requests/{request_id}/comments/{comment_id}`
+
+**Parameters**
+
+| Name       | Type   | Location | Description                                                  |
+| ---------- | ------ | -------- | ------------------------------------------------------------ |
+| `request_id`   | string | path     | The request's public identifier. |
+| `comment_id`       | string | path     | The comment's public identifier                |
+| `payload`    | object | body     | The data associated with he comment. See above. |
+| `accept`   | string | header   | - `application/json` (default)<br />- `application/vnd.inveniordm.v1+json` (not yet) |
+
+**Request**
+
+```http
+PUT /api/requests/{request_id}/comments/{comment_id} HTTP/1.1
+Content-Type: application/json
+
+{
+  "payload": {
+    "content": "I would use these subject terms to align this record and the other one with others in the community.",
+    "format": "html"
+  }
+}
+```
+
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+
+{
+  "created": "2020-11-27 10:52:23.945755",
+  "id": "{comment_id}",
+  "links": {
+    "self": "{scheme+hostname}/api/requests/{rquest_id}/comments/{comment_id}"
+  },
+  "payload": {
+    "content": "I would use these subject terms to align this record and the other one with others in the community.",
+    "format": "html"
+  },
+  "revision_id": 2,
+  "type": "C",
+  "updated": "2020-11-27 10:55:23.969244",
+}
+```
+
+### Delete a comment
+
+`DELETE /api/requests/{request_id}/comments/{comment_id}`
+
+Deleting a comment will clear its content and mark it as a "deleted comment event". You can only delete your own comments unless you are a request's reviewer or admin.
```

Comment:
I think for now, we are letting the receivers delete comments. But even with flagging feature, it might still make sense to let the receivers delete comments and moderate the exchange themselves without needing an admin. I think we will revisit this once we use it on the UI.